src/main/java/org/acra/builder/ReportExecutor.java

```diff
@@ -145,6 +146,7 @@ public void execute(@NonNull final ReportBuilder reportBuilder) {
                 || (config.resToastText() != 0 && (reportingInteractionMode == ReportingInteractionMode.NOTIFICATION || reportingInteractionMode == ReportingInteractionMode.DIALOG));
 
         final TimeHelper sentToastTimeMillis = new TimeHelper();
+        boolean displayingToast = false;
```

Comment:
Why introduce the 'displayingToast' variable?
Surely you just want to make sure you don't kill the process if the debugger is connected.